### DISCUSSION TITLE
support custom footer.html

### DISCUSSION THREAD

**Discussion by lucays** [pradyunsg/furo#392] [Ideas] [answered]
Hi, how about support custom footer.html like RTD theme?

I need it to add custom comment feature. Now I only can do it by install my forked repo.
My change is simple: https://github.com/lucays/furo/commit/f02c66fb0ec7e558e204f2c71bca1d1483eb62ff

If you want to see the effect after adding the feature:
    my project use it is in: https://github.com/lucays/ocg-rule/tree/dev/docs/.templates
    after add custom comment feature, the web page: https://ocg-rule.readthedocs.io/zh_CN/latest/c06/2022.html#id163 (Please slide to the bottom)

The custom comment need load js, sometimes have problems. You can also see the screenshot
![image](https://user-images.githubusercontent.com/29814851/156596625-3d64220a-76dd-470b-91cb-0e89f3ed9296.png)

If ok, can I make a pull request?

**Comment by pradyunsg** [comment, marked as answer]
> Hi, how about support custom footer.html like RTD theme?

I'm not really interested in supporting arbitrary code injection into the theme. It's more complexity for me to worry about supporting, and... well, as I said, I'm not interested in doing that.

While I'm sure that many of the folks who make such customisations will make them in a sensible way and be generally understanding that the license literally states "No warranties", I don't want to go out of my way to provide a mechanism to extend this theme in ways I'm not interested in supporting. If you'd like that, please feel welcome to fork and extend this theme with your customisations!

**Reply by lucays** [reply]
well, thank you for your reply
